samples/teams-sso-node/client/index.html

```diff
@@ -0,0 +1,55 @@
+<!-- Copyright (c) Microsoft Corporation.
+Licensed under the MIT License. -->
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+  <title>Teams SSO Sample for Microsoft Graph Toolkit in Node.js</title>
+
+  <link rel="shortcut icon" href="g-raph.png">
+  <script src="https://unpkg.com/@microsoft/teams-js/dist/MicrosoftTeams.min.js" crossorigin="anonymous"></script>
+  <script src="./mgt/mgt-loader.js"></script>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+  <main id="main-container" role="main" class="container">
+    <!-- <mgt-teams-sso-provider client-id="{ YOUR CLIENT ID }"
```

Comment:
`mgt-teams-provider` instead of `mgt-teams-sso-provider`

Reply:
Also add a comment that this is an alternative way to initialize the provider

Reply:
I've added a comment in the index.html page. Above the provider component and also a comment before the "code".